test/AquaThermAire.txt

```diff
@@ -0,0 +1,71 @@
+verbosity silent
+numNodes 1 #number of nodes
+setpoint 125 F
+volume 54.4 gal
+UA 10.35 kJperHrC
+depressTemp false
+mixOnDraw true
```

Comment:
This should be false, the draw doesn't go into the tank at all, and it shouldn't cause any mixing of the water.

Reply:
done